commitlint/testcommitlint.sh

```diff
@@ -440,5 +440,16 @@ word
 Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>"
 
 
+assert "true" \
+       "Commit with lowercase entries" \
+       "fix: Commit with multiple multiline Changelogs
```

Comment:
```suggestion
       "fix: Commit with lowercase changelog
```

Something like that represents better what the test is about

Reply:
Omg, so obvious! :see_no_evil: Thanks!